posts/exploring-iot-infrastructure-support-in-aws-part2.md

```diff
@@ -0,0 +1,450 @@
+---
+authors:
+- Ben Edridge
+tags:
+- AWS
+- IoT
+date: 2018-07-19T11:41:57.000Z
+title: "Exploring AWS IoT Core and Greengrass Offerings (Part 2)"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/07/aws_iot_esp_device.jpeg
+---
+
+This a second post following from the [part one](https://blog.ippon.tech/exploring-iot-infrastructure-support-in-aws-part1) of "Exploring AWS IoT Core and Greengrass Offerings". In part one I discussed the background and theoretical aspects of the AWS IoT offerings including [AWS IoT Core](https://aws.amazon.com/iot-core/), [Greengrass](https://docs.aws.amazon.com/greengrass/latest/developerguide/what-is-gg.html#gg-platforms) and [Amazon FreeRTOS](https://aws.amazon.com/freertos/). I also introduced a proof of concept which will be further discussed in this blog post. More example code will be provided and the deployment options will be evaluated.
+
+# The Journey
+
+It's always been a hobby of mine to explore low-powered/mobile computing devices and hack around. Whether it was revitalising old routers, increasing page limits for printers or customising the Android OS. The basic goal for me was I wanted to have some fun whilst learning about the software and the supporting OS. The questions I always had were:
```

Comment:
> The questions I always had were:

I think this should be reworded. Maybe something like `Everytime I pick up one of these devices, I ask myself the following questions`